api/v1alpha1/webhook.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+// Maximum length of bundle name
+const maxNameLength = 40
+
+// log is for logging in this package.
+var bundlelog = logf.Log.WithName("bundle-resource")
+
+func (r *Bundle) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	bundlelog.Info("SetupWebhookWithManager")
```

Comment:
nit: I think we can drop this log statement?

Reply:
Yes, I'll take it out.